corehq/ex-submodules/auditcare/management/commands/gdpr_scrub_user_auditcare.py

```diff
@@ -22,6 +22,6 @@ def update_username(event_dict):
             event_dict['user'] = new_username
             return DocUpdate(doc=event_dict)
 
-        new_username = "Redacted User (GDPR)"
+        new_username = "Redacted User (Right To Forget)"
```

Comment:
I agree that typing it out in both places isn't great, but `gdpr_scrub_user_auditcare` and `gdpr_scrub_user_from_forms` are two separate management commands (they're both for GDPR compliance, but aren't combined into one management command because both aren't necessarily run when a user requests an audit), is there there a good place to put `NEW_USERNAME` where both files would see it?

Reply:
How about https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/users/const.py

Reply:
Perfect, moved the variable there and renamed it `REDACTED_USERNAME`

https://github.com/dimagi/commcare-hq/pull/20387/commits/b4fec5a64589f66d6f8c9c03c8be83f499e15b81